packages/events-example/src/deployments/pulumi.ts

```diff
@@ -0,0 +1,94 @@
+
+import { CliUx } from "@oclif/core";
+import { 
+  DestroyResult,
+    InlineProgramArgs, 
+    LocalWorkspace, 
+    LocalWorkspaceOptions, 
+    PulumiFn, 
+    Stack, 
+    StackSummary, 
+    UpResult
+} from "@pulumi/pulumi/automation";
+import { Instance } from "@theniledev/js";
+
+export interface PulumiFnGen {
+  (something: any): PulumiFn
```

Comment:
is this something something useful?

Reply:
Yeah, it's a light wrapper around that `PulumiFn` that @norwood had originally labeled as a `pulumiProgram`. That `something` is actually content that gets embedded in the index.html in the S3 bucket that gets deployed, which includes some of the Nile instance data.

Reply:
Maybe it'd be helpful to rename `something` to `content` or similar.